### DISCUSSION TITLE
Yugabyte Database

### DISCUSSION THREAD

**Discussion by alberk8** [JasperFx/marten#3263] [Q&A]
Have anyone have experience running Marten against Yugabyte YSQL (Postgres 11 compatible) Database?

**Comment by mysticmind** [comment]
I did setup YugaByte and tried out running the Marten test suite. It resulted in several errors with YugaByte as below albeit it was inserting db entries etc:

```
---- Npgsql.PostgresException : 40001: could not serialize access due to concurrent update (query layer retry isn't possible because data was already sent, if this is the read committed isolation (or) the first statement in repeatable read/ serializable isolation transaction, consider increasing the tserver gflag ysql_output_buffer_size)
```

Looking at the above, it does not look like it is that much compatible. So not worthwhile to do anything further.
